protocol/examples/reqres/Cargo.toml

```diff
@@ -6,7 +6,9 @@ edition = "2018"
 
 [dependencies]
 async-std ="1.9.0"
+bincode = "1.3.3"
 libp2p = " 0.38.0"
+libp2p-yamux = "0.32.0"

```

Comment:
Looks like yamux crate available via `libp2p`: https://docs.rs/libp2p/0.38.0/libp2p/yamux/struct.Yamux.html, no needs to add it as dependency.

Reply:
without it i have build error

177 |         .multiplex(libp2p_yamux::YamuxConfig::default())
    |                    ^^^^^^^^^^^^ use of undeclared crate or module `libp2p_yamux`

Maybe I building in different way then expected. 
In may case to avoid build all and build only example:
C:\ws\yamukha\robonomics\protocol\examples\reqres>cargo build --release

Or how you recommend to build?

Reply:
just change libp2p_yamux to libp2p::yamux